cmd/generator/main.go

```diff
@@ -139,3 +147,16 @@ func main() { // nolint:gocyclo
 	}
 	fmt.Printf("\nGenerated %d resources!\n", count)
 }
+
+func getRegionSchema() (*schema.Schema, error) {
+	c := "Region is the region you'd like your resource to be created in.\n"
+	comment, err := comments.New(c, comments.WithTFTag("-"))
```

Comment:
Not related to this PR at all. Just noticed that we're returning error in `comments.New` and it felt weird since unrecognized lines are skipped. Seems like it's because `ParseAsTerrajetOption` can return error but it seems it can also just return [`false`](https://github.com/crossplane-contrib/terrajet/blob/0a3e0f1607e63bcd0836b4b9bd6093ed09fe5b12/pkg/markers/terrajet.go#L58) for that case. I can change that later if you're onboard.

Reply:
In fact, that was the case in the beginning. `ParseAsTerrajetOption` was not returning error but only returning whether it could parse or not. This is all good for `kubebuilder` or `crossplane tools` markers but not a good idea if it is (intended to be) a terrajet marker.
But later changed my mind, since I didn't feel good about silently ignoring a marker line that starts with `+terrajet:`. This could be a possible scenario when a provider author is trying to use a non-supported terrajet marker which was added in a newer version of terrajet. Returning an error in that case, sounds better than silently ignoring. 
However, just realized that :), our API is not using marker string directly, but uses the same `comments.New` with the corresponding option which invalidates the above possibility in fact. 
So, going for such a simplification based on this assumption (no one will write terrajet markers by hand), sounds good to me. We can change it in a later PR.